src/components/ImageSlider/ImageSlider.js

```diff
@@ -0,0 +1,110 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import { Button, Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const ImageSlider = ({ images, height }) => {
+  const [selectedImageIndex, setSelectedImageIndex] = useState(0);
+
+  const handleClickNext = () => {
+    const nextImageIndex = selectedImageIndex + 1;
+    setSelectedImageIndex(nextImageIndex % images.length);
+  };
+
+  const handleClickPrev = () => {
+    const prevImageIndex = selectedImageIndex - 1;
+    if (prevImageIndex < 0) {
+      setSelectedImageIndex(images.length - 1);
+    } else {
+      setSelectedImageIndex(prevImageIndex % images.length);
+    }
+  };
+
+  return (
+    <div
+      css={css`
+        position: relative;
+        margin-bottom: 2rem;
+      `}
+    >
+      {images.length > 1 && (
```

Comment:
Rather than checking here, on the anchor tag, and on the image, it might make more sense to do a check if we have images and supply an early return:

```jsx
if (!images.length) {
  // empty state (no buttons, no link, etc.)
}

return (
  // ...other code
);
```

Reply:
not sure i quite follow. i just wanted to only add nav arrows / dots if we have more than one image to navigate through. 

i wasn't 100% sure if i should account for no images here (and thus a placeholder image) or if it was safe to assume we would do that via the `src/templates/ObservabilityPackDetails.js` since we likely have to do some checks there to see what's in the data. ¯\_(ツ)_/¯